.github/workflows/build.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
   build:
     strategy:
       matrix:
-        go-version: [1.16.x]
+        go-version: [1.17.x]
```

Comment:
why 17 instead of 18?

Reply:
Glad you asked. This shouldn't have any direct meaning, but I'm trying to move things gradually to 1.17 without getting warnings about not being able to build a module due to a dependency forcing a lower version.

Thanks for taking a look.